### PR TITLE
Improve build instructions and other cleanups

### PR DESCRIPTION
**Pull request by MartinGuehmann**
This PR improves the readme of this repository, to include better build instructions for Linux and Windows. It also cleans up the Visual Studio configuration Debug Browse, so that it does not clash with the Debug configuration. It adds more files to .gitignore, so that the a lot of files that are generated by Visual Studio, the game, and that are added to complete the working copy, do not clutter git status and won't be added, accidentally. Plus some other minor cleanup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,20 +11,51 @@
...
+
+If you have the GoG version, use `innoextract` to extract all the required files (you can `sudo apt install innoextract` if you need) from the `setup_call_to_power2_2.0.0.13.exe` windows installer.
+For example `innoextract -m -I app  setup_call_to_power2_2.0.0.13.exe` will extract the `app`folder that contains all the game files, which are in the same subfolders as in the CD version: `ctp2_data`, `ctp2_program` and `Scenarios`. Copy these folders over your working copy without replacing files that already exist. Now you should have those directories in your working copy.
+
+* `ctp2_code`
+* `ctp2_data`
+* `Scenarios`
+* `ctp2_program`
+
+The folder `ctp2_program` is a stripped down version of `ctp2_code` without the source code and is the version that was shipped with the game. It is not needed for compiling and running the game. If it is an from a previous installation and you have some save games you want to use in your working copy then copy the folder `/ctp2_program/ctp/save/` and all its content to `/ctp2_code/ctp/save/`.
```
Comment (LynxAbraxas) [inline README.md:48]: Seems something got lost here: If it is an from ....

File: .gitignore [hunk 2]
```diff
@@ -260,4 +267,477 @@ ctp2_code/libs/freetype-1.3.1/test/ftstrtto
 ctp2_code/libs/freetype-1.3.1/test/fttimer
 ctp2_code/libs/freetype-1.3.1/test/ftview
 ctp2_code/libs/freetype-1.3.1/test/ftzoom
-ctp2_code/ctp2
+ctp2_code/ctp/ctp2
+ctp2_data/default/graphics/cursors/UC001.tga
```
[the remark this answers is not in the document]
Comment (MartinGuehmann) [reply .gitignore:271]: These files come from the retail version, I don't think we will ever modify them. Basically if you replace them you would modify the look and feel of the interface, and would end up  with something else. I doubt that we would ever modify the original sprites and replace the original artwork that is zipped in the pic5*5.zfs files.

On the other hand as far as I can see you can add files anyway even so they are ignored. And new files should have new names that aren't listed anyway. I just saw that also these folders are on the list:

ctp2_data/default/graphics/icons/
ctp2_data/default/graphics/patterns/

Those shouldn't be there.

File: README.md (hunk 1 above)
Comment (MartinGuehmann) [reply README.md:48]: Indeed something is missing here.

**Comment by LynxAbraxas** [comment]
@MartinGuehmann: Many thanks for these changes, look fine to me. 
When merging this, please also consider quick changes to the GH-settings of the game as suggested here:#266.

File: .gitignore (hunk 2 above)
[the remark this answers is not in the document]
Comment (LynxAbraxas) [reply .gitignore:271]: Perhaps just add a comment line above all those mentioning that the entry should be removed if a new version is provided in a commit. 

> On the other hand as far as I can see you can add files anyway even so they are ignored. 

Yes, but that is quite dirty and bad practice, if added it should be removed from the .gitignore and that change added to the same commit.

**Comment by MartinGuehmann** [comment]
Then merge this.

**Pull request merged**